diff --git a/cmake/catalyst-macros.cmake b/cmake/catalyst-macros.cmake
index f1e700bde155..d60e41519014 100644
--- a/cmake/catalyst-macros.cmake
+++ b/cmake/catalyst-macros.cmake
@@ -69,6 +69,7 @@ function (catalyst_implementation)
   else ()
     set(catalyst_impl_catalyst_target_req "${catalyst_impl_CATALYST_TARGET}")
   endif ()
+
   get_property(catalyst_conduit_property_is_external_is_set
     TARGET    "${catalyst_impl_catalyst_target_req}"
     PROPERTY  CATALYST_WITH_EXTERNAL_CONDUIT
@@ -81,12 +82,12 @@ function (catalyst_implementation)
       message(AUTHOR_WARNING
         "The `CATALYST_TARGET` '${catalyst_impl_CATALYST_TARGET}' does not have "
         "the `CATALYST_WITH_EXTERNAL_CONDUIT` target property set; assuming "
-        "that an internal Conduit is used.")
+        "that an external Conduit is used.")
     endif ()
     set_property(GLOBAL
       PROPERTY
         "${catalyst_warning_property_name}" 1)
-    set(catalyst_impl_conduit_is_external 0)
+    set(catalyst_impl_conduit_is_external 1)
   else ()
     get_property(catalyst_conduit_property_is_external
       TARGET    "${catalyst_impl_catalyst_target_req}"
