diff --git a/configure.ac b/configure.ac
index 5b656838cfa7..a11f6c023e54 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,7 +40,7 @@ LT_INIT
 
 PKG_CHECK_MODULES([LIBCONFIG], [libconfig >= 1.5])
 PKG_CHECK_MODULES([LIBUV], [libuv >= 1.18])
-PKG_CHECK_MODULES([LIBFUSE], [fuse >= 2.9.7])
+PKG_CHECK_MODULES([LIBFUSE], [fuse3 >= 3.0])
 PKG_CHECK_MODULES([LIBYAML], [yaml-0.1 >= 0.1.7])
 PKG_CHECK_MODULES([LIBNL], [libnl-3.0])
 PKG_CHECK_MODULES([LIBNUMA], [numa >= 2.0])
